src/Config.php

```diff
@@ -51,9 +51,14 @@ public function __construct($config)
     /** @var string Password to use for authentication */
     public $password;
 
+    /** @var string */
+    public $type;
```

Comment:
`type` comes from the `resources.ini` of Icinga Web and is not a valid configuration option for ipl SQL connections. I would remove the property and silently discard all key-value pairs where `Config` has no matching property. This should also be documented then.